{
  "uid" : "6f81bc8adf2253ab",
  "name" : "LoginHataliParola",
  "fullName" : "Login.LoginHataliParolaTest.LoginHataliParola",
  "historyId" : "d9c33f9dd65deaf5301d616b38c91f0c",
  "time" : {
    "start" : 1669733733554,
    "stop" : 1669733733556,
    "duration" : 2
  },
  "description" : "Kullancı Adı ve Yanlış Şifre ile giriş yapma senaryosu.",
  "descriptionHtml" : "<p>Kullancı Adı ve Yanlış Şifre ile giriş yapma senaryosu.</p>\n",
  "status" : "skipped",
  "statusMessage" : "The local appium server has not been started. The given Node.js executable: C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium\\Appium.exe Arguments: [C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium, --port, 11922, --address, 0.0.0.0] \n",
  "statusTrace" : "io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium\\Appium.exe Arguments: [C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium, --port, 11922, --address, 0.0.0.0] \n\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:164)\r\n\tat Libraries.BaseClass.startAppiumServer(BaseClass.java:78)\r\n\tat Libraries.BaseClass.setUp(BaseClass.java:127)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)\r\n\tat org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\nCaused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:11922/wd/hub/status] to be available after 120005 ms\r\n\tat org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:129)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:151)\r\n\t... 27 more\r\nCaused by: java.util.concurrent.TimeoutException\r\n\tat java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)\r\n\tat com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:156)\r\n\tat org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)\r\n\t... 29 more\r\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "setUp",
    "time" : {
      "start" : 1669733612710,
      "stop" : 1669733733084,
      "duration" : 120374
    },
    "description" : "",
    "status" : "broken",
    "statusMessage" : "The local appium server has not been started. The given Node.js executable: C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium\\Appium.exe Arguments: [C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium, --port, 11922, --address, 0.0.0.0] \n",
    "statusTrace" : "io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium\\Appium.exe Arguments: [C:\\Users\\uyildiz\\AppData\\Local\\Programs\\Appium, --port, 11922, --address, 0.0.0.0] \n\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:164)\r\n\tat Libraries.BaseClass.startAppiumServer(BaseClass.java:78)\r\n\tat Libraries.BaseClass.setUp(BaseClass.java:127)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)\r\n\tat org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\nCaused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:11922/wd/hub/status] to be available after 120005 ms\r\n\tat org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:129)\r\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:151)\r\n\t... 27 more\r\nCaused by: java.util.concurrent.TimeoutException\r\n\tat java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)\r\n\tat com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:156)\r\n\tat org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)\r\n\t... 29 more\r\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true
  } ],
  "afterStages" : [ {
    "name" : "stopAppiumAndDriver",
    "time" : {
      "start" : 1669733733568,
      "stop" : 1669733733569,
      "duration" : 1
    },
    "description" : "",
    "status" : "broken",
    "statusMessage" : "java.lang.NullPointerException",
    "statusTrace" : "java.lang.NullPointerException\r\n\tat Libraries.BaseClass.stopAppiumAndDriver(BaseClass.java:138)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)\r\n\tat org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true
  } ],
  "labels" : [ {
    "name" : "package",
    "value" : "Login.LoginHataliParolaTest"
  }, {
    "name" : "testClass",
    "value" : "Login.LoginHataliParolaTest"
  }, {
    "name" : "testMethod",
    "value" : "LoginHataliParola"
  }, {
    "name" : "parentSuite",
    "value" : "Default Suite"
  }, {
    "name" : "suite",
    "value" : "ttPay_POM"
  }, {
    "name" : "subSuite",
    "value" : "Login.LoginHataliParolaTest"
  }, {
    "name" : "host",
    "value" : "N110688"
  }, {
    "name" : "thread",
    "value" : "22032@N110688.main(1)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "severity",
    "value" : "minor"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "6f81bc8adf2253ab.json",
  "parameterValues" : [ ]
}